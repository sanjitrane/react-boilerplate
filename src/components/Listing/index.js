import React, { Component } from 'react';
import { connect } from 'react-redux';
import { Link } from 'react-router-dom';

class Listing extends Component {

  
  constructor(props){
    super(props)
  }
  
  componentDidMount() {
  
  }


  render() {

    return(
      <div className="container">
        <h1>Listing page</h1>
      </div>
    );
  }
};

const mapStateToProps = state => state
export default connect(mapStateToProps)(Listing);