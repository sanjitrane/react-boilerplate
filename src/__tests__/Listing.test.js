import React from 'react';
import renderer from 'react-test-renderer';
import { shallow} from 'enzyme';
import Listing from '../components/Listing';
import configureMockStore from 'redux-mock-store';
import Thunk from 'redux-thunk'

const middleware = [Thunk]
const mockStore = configureMockStore(middleware);
const store = mockStore()

describe('Tests Listing component', ()=>{
    const wrapper = shallow(<Listing/>, {context:{store}})
    it('renders properly',()=>{
        shallow(<Listing/>, {context:{store}})
        expect(wrapper).toMatchSnapshot()
    });
});