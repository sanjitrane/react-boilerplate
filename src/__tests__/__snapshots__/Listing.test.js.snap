// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests Listing component renders properly 1`] = `ShallowWrapper {}`;
